src/Microsoft.VisualStudio.Services.Agent/ProcessInvoker.cs

```diff
@@ -213,6 +222,13 @@ public sealed class ProcessInvoker : AgentService, IProcessInvoker
                 _proc.StandardInput.Dispose();
             }
 
+            if(!waitForExit)
```

Comment:
removed the changes. I thought of using the common library for my process related operations but yes I agree my need is very minimal and just Process.Start() can be right thing.